### PR TITLE
Moves some hard values into the config

### PR DESCRIPTION
**Pull request by darrylmasson**
It's generally better than using magic numbers.

### CODE REVIEW

File: dispatcher/config.ini [hunk 1]
```diff
@@ -36,6 +36,13 @@ RetryReset = 3
 # Min time between Stop-Arm and Arm-Start commands
 TimeBetweenCommands = 6
 
+# Time between reader and CC start
+# THIS IS AN IMPORTANT VALUE, DON'T CHANGE IT UNLESS YOU KNOW WHAT YOU'RE DOING
+# 20210308 - Darryl
+StartCmdDelay = 1
```
Comment (jorana) [inline dispatcher/config.ini:42]: Why the change from 1.5 to 1?

File: dispatcher/DAQController.py [hunk 2]
```diff
@@ -43,6 +43,8 @@ def __init__(self, config, mongo_connector, log, hypervisor):
         self.time_between_commands = int(config['TimeBetweenCommands'])
         self.can_force_stop={k:True for k in detectors}
         self.one_detector_arming = False
+        self.start_cmd_delay = int(config['StartCmdDelay'])
```
Comment (jorana) [inline dispatcher/DAQController.py:46]: why int, why not float? 1.5 would not be possible which makes it a little unclear

File: dispatcher/DAQController.py [hunk 3]
```diff
@@ -284,20 +286,20 @@ def control_detector(self, command, detector, force=False):
...
                 # we can safely short the logic here and buy an extra logic cycle
                 self.one_detector_arming = False
-                delay = 1.5
+                delay = self.start_cmd_delay
             else: # stop
                 readers, cc = self.mongo.get_configured_nodes(detector,
                     self.goal_state['tpc']['link_mv'], self.goal_state['tpc']['link_nv'])
                 hosts = (cc, readers)
                 if force or self.latest_status[detector]['status'] not in [DAQ_STATUS.RUNNING]:
                     delay = 0
```
Comment (jorana) [inline dispatcher/DAQController.py:295]: Don't know if it makes sense also to add a arm delay or a stop delay to the config?

File: dispatcher/DAQController.py (hunk 2 above)
Comment (jorana) [inline dispatcher/DAQController.py:46]: I personally would put this in a dictionary rather than two (or possibly more) growing list of delays. For two it doesn't matter but as soon as we have >3 parameters, I would use a dict.

**Review by jorana — APPROVED** [review]
Looks good, just some minor considerations

File: dispatcher/config.ini (hunk 1 above)
Comment (darrylmasson) [reply dispatcher/config.ini:42]: 1 was the intended value originally. I changed it to 1.5 after The Incident to be extra safe while we dug into what happened. In principle it can be as low as 105ms.

File: dispatcher/DAQController.py (hunk 2 above)
Comment (darrylmasson) [reply dispatcher/DAQController.py:46]: We could

File: dispatcher/DAQController.py (hunk 3 above)
Comment (darrylmasson) [reply dispatcher/DAQController.py:295]: We could, but we haven't found a reason to stagger ARMs. We only stagger STOPs if the readout is live so the end of the datastream doesn't get corrupted, otherwise everything gets it at the same time.

File: dispatcher/DAQController.py (hunk 2 above)
Comment (darrylmasson) [reply dispatcher/DAQController.py:46]: For sure, but we only have two values. If we decide on a reason to include more then a dict becomes a better idea.
